pkg/coreapi/graph/resolvers/events.go

```diff
@@ -20,7 +21,14 @@ func (r *queryResolver) Event(ctx context.Context, query models.EventQuery) (*mo
 	}
 
 	evt := evts[0]
+
 	createdAt := time.UnixMilli(evt.Timestamp)
+	if evt.Timestamp == 0 {
```

Comment:
Is it still a bug that `evt.Timestamp` isn't defined here?

Reply:
It is - that needs to be fixed with additional unit tests in a followup PR